kubernetes/samples/scripts/common/domain-template.yaml

```diff
@@ -34,6 +34,8 @@ spec:
   %LOG_HOME_ON_PV_PREFIX%logHomeEnabled: %LOG_HOME_ENABLED%
   # The in-pod location for domain log, server logs, server out, and Node Manager log files
   %LOG_HOME_ON_PV_PREFIX%logHome: %LOG_HOME%
+  # The in-pod location for data storage of default and custom file stores files
```

Comment:
Maybe we can provide a more detail here? For example, change "The" to "Optional", and add a sentence something like: "If dataHome is not set or is set to an empty string, data storage directories are determined from WebLogic domain home configuration.".

Reply:
Confirmed resolved.